content/en/examples/access/certificate-signing-request/clusterrole-sign.yaml

```diff
@@ -21,7 +21,7 @@ rules:
   - certificates.k8s.io
   resources:
   - signers
-  resourceName:
+  resourceNames:
```

Comment:
Great catch. Verified this is correct using https://kubernetes.io/docs/reference/access-authn-authz/rbac/